physionet-django/physionet/settings/settings.py

```diff
@@ -1,45 +1,47 @@
 import os
+import pathlib
 
 from decouple import config
 
 from physionet.settings.base import *
 
-DEBUG = False
-
-ALLOWED_HOSTS = ['physionet-production.ecg.mit.edu', 'physionet.org', 'www.physionet.org']
-SITE_ID = 3
+ALLOWED_HOSTS = config("ALLOWED_HOSTS").split(',')
+SITE_ID = config("SITE_ID")
 
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': 'physionet',
-        'USER': 'physionet',
-        'PASSWORD': config('DATABASES_PASSWORD'),
-        'HOST': 'localhost',
+        'NAME': config("DB_NAME"),
+        'USER': config("DB_USER"),
+        'PASSWORD': config('DB_PASSWORD'),
+        'HOST': config("DB_HOST"),
         'PORT': '',
+        'TEST': {
+            'MIRROR': 'default'
+        }
     }
 }
 
 # When ready, use the following:
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
-EMAIL_HOST = 'localhost'
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend' if DEBUG else 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = config("EMAIL_HOST")
 EMAIL_PORT = 25
 EMAIL_HOST_USER = ''
 EMAIL_HOST_PASSWORD = ''
 EMAIL_USE_TLS = False
 
 EMAIL_FROM_DOMAINS = ['physionet.org']
-DEFAULT_FROM_EMAIL = 'PhysioNet Automated System <noreply@physionet.org>'
-CONTACT_EMAIL = 'PhysioNet Contact <contact@physionet.org>'
-SERVER_EMAIL = 'PhysioNet System <root@physionet.org>'
+DEFAULT_FROM_EMAIL = config("DEFAULT_FROM_EMAIL")
```

Comment:
It would be great to use old values as default ones, not to break compatibility for physionet's existing deployments